pkg/controller/kogitoapp/resource/build_config_s2i.go

```diff
@@ -36,6 +36,7 @@ const (
 	buildS2IlimitCPUEnvVarKey    = "LIMIT_CPU"
 	buildS2IlimitMemoryEnvVarKey = "LIMIT_MEMORY"
 	mavenMirrorURLEnvVar         = "MAVEN_MIRROR_URL"
+	mavenDownloadOutput          = "MAVEN_DOWNLOAD_OUTPUT"
```

Comment:
change to `mavenDownloadOutputEnvVar`

Reply:
chnaged name to mavenDownloadOutputEnvVar